mmcv/runner/hooks/logger/wandb.py

```diff
@@ -39,7 +57,23 @@ def before_run(self, runner):
             self.wandb.init(**self.init_kwargs)
         else:
             self.wandb.init()
-
+        
+        if self.config_path is not None:
+            shutil.copy2(self.config_path, self.wandb.run.dir)
```

Comment:
Can the `config_path` also be a path of a directory?

Reply:
Note that `shutil.copy2` only works for copying a file rather than a directory.

Reply:
You are right. I should add a line to copy a tree for files such as `shutil.copytree`. 
I thought about just case of hyperparam tunning, but I got a recommendation from my peer about the usage of config dirs. In general case, many users want to manage their configurations in a separated from such as model.py, optimizer.py, logger.py etc.

Reply:
In the under commit, I use copy_tree function in distutils.dir_util because shutil.copytree doesn't work with an existing path. In recent versions of shutil.copytree have an argument to work with existing path but not for old python.